src/hooks/useResource.js

```diff
@@ -18,7 +18,10 @@ import {
 } from "actions/resources"
 import { useHistory } from "react-router-dom"
 
-const useResource = (errorKey, errorRef) => {
+const useResource = (
+  errorKey,
+  { resourceTemplateId = null, resourceURI = null }
```

Comment:
useResource is bound to a particular resourceTemplateId / resourceURI now, rather than providing handle functions in which these are passed in as arguments.